publications-2018.bib

```diff
@@ -1,74 +1,5 @@
 % Encoding: US-ASCII
 
-@MastersThesis{2016:c--a--h--blom:state-of-the-art-numerical-subduction-modelling-with-aspect--thermo-mechanically-coupled-viscoplastic-compressible-rheology--free-surface--phase-changes--latent-heat-and-open-sidewalls,
-  author  = {{C. A. H. Blom}},
-  school  = {Utrecht University},
-  title   = {{State of the art numerical subduction modelling with ASPECT; thermo-mechanically coupled viscoplastic compressible rheology, free surface, phase changes, latent heat and open sidewalls}},
-  url     = {https://dspace.library.uu.nl/handle/1874/348133},
-  year    = 2016
-}
-
-@Misc{2017:badia.martin.ea:fempar,
-  title   = {FEMPAR: An object-oriented parallel finite element framework},
-  author  = {Santiago Badia and Alberto F. Mart{\'i}n and Javier Principe},
-  year    = 2017,
-  eprint  = {1708.01773},
-  archiveprefix = {arXiv},
-  primaryclass = {cs.CE}
-}
-
-@Article{2017:emerson.farrell.ea:computing,
-  doi     = {10.1080/02678292.2017.1365385},
-  url     = {https://doi.org/10.1080/02678292.2017.1365385},
-  year    = 2017,
-  month   = aug,
```

Comment:
Here and for the following: The journals have been published in 2018, but they have been online already in 2017.